components/Modal/CreateImageUpload.js

```diff
@@ -442,88 +496,297 @@ class CreateImageUploadModal extends React.Component {
+                </label>
+                <Popover
+                  id="image-name-popover"
+                  bodyContent={
+                    <React.Fragment>
+                      <FormattedMessage defaultMessage="Provide a file name to be used for the image file that will be uploaded." />
+                    </React.Fragment>
+                  }
+                  aria-label="image name help"
                 >
-                  <TextInput
-                    value={this.state.uploadSettings[key] || ""}
-                    type={providerSettings[provider].settings[key].isPassword ? "password" : "text"}
-                    id={key}
-                    key={key}
-                    onChange={this.setUploadSettings}
-                    isRequired
-                  />
-                </FormGroup>
-              ))}
-            </Form>
-          )}
+                  <Button variant="plain" aria-label="image name help">
+                    <OutlinedQuestionCircleIcon id="popover-icon" />
+                  </Button>
+                </Popover>
+              </div>
+              <TextInput
+                className="pf-c-form-control"
+                value={imageName}
+                type="text"
+                id="image-name-input"
+                onChange={this.setImageName}
+              />
+            </div>
+            <div className="pf-c-form__group">
+              <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+                <label htmlFor="bucket-input" className="pf-l-flex__item">
+                  <span className="pf-c-form__label-text">Amazon S3 bucket</span>
+                  <span className="pf-c-form__label-required" aria-hidden="true">
+                    &#42;
+                  </span>
+                </label>
+                <Popover
+                  id="bucket-popover"
+                  bodyContent={
+                    <React.Fragment>
+                      <FormattedMessage
+                        defaultMessage="
+                          Provide the S3 {bucket} name to which the image file will be uploaded before being imported into EC2. 
+                          The {bucket} must already exist in the Region where you want to import your image. You can find a list of {buckets} on the
+                          {bucketsPage} page in the {amazon} S3 storage service in the AWS console.
+                        "
+                        values={{
+                          bucket: "bucket",
+                          buckets: "buckets",
+                          bucketsPage: <strong>S3 buckets</strong>,
+                          amazon: "Amazon"
+                        }}
+                      />
+                    </React.Fragment>
+                  }
+                  aria-label="bucket help"
+                >
+                  <Button variant="plain" aria-label="bucket help">
```

Comment:
I usually recommend that the help text button follow this format: "[field label] help". In this case, I can see the case for shortening it when input labels are really long, but am wondering if we should at least include "S3" in front of "bucket help".